_drafts/making-backend-scalable.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Making Autolab's Backend Scalable"
+author: Ilter Canberk
+
+tags: [autolab, community, backend, scalability]
```

Comment:
Makes sense. Btw I made a lot of changes on this draft but not able to push
right now, so it's not really worth reviewing.
On Fri, Mar 27, 2015 at 20:59 Jake Zimmerman <notifications@github.com>
wrote:

> In _drafts/making-backend-scalable.markdown
> <https://github.com/autolab/autolab.github.io/pull/1#discussion_r27321909>
> :
>
> > @@ -0,0 +1,70 @@
> > +---
> > +layout: post
> > +title: "Making Autolab's Backend Scalable"
> > +author: Ilter Canberk
> > +
> > +tags: [autolab, community, backend, scalability]
>
> I would say that fewer tags is better, and also that tagging specific
> things is better. I would probably tag this just Tango, Redis, and maybe
> scalability .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/autolab/autolab.github.io/pull/1/files#r27321909>.
>